assets/scss/style.scss

```diff
@@ -0,0 +1,246 @@
+// // Typografhy
+@import url("https://fonts.googleapis.com/css2?family=Chivo:wght@300&display=swap");
+
+// // Colors
+
+$green: #54e6af;
+$darkGray: #2c344b;
+$black: #121725;
+$white: #ffffff;
+$lightGray: #5a668a;
+$silver: #c2cbe5;
+$warning: #fb3e3e;
+
+$backgroundMobile: rgba(18, 23, 37, 0.9);
+
+$fontInput: rgba(255, 255, 255, 0.5);
+$buttonHover: #b3ffe2;
+
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Chivo", sans-serif;
+  background: $black;
+  .container {
+    background: linear-gradient($backgroundMobile, $backgroundMobile),
+      url("/assets/mobile/image-host.jpg");
+    background-repeat: no-repeat;
+    .logo {
+      text-align: center;
+      margin: 62px auto 0px auto;
+    }
+    main {
+      padding: 57px 24px 94px 24px;
+      display: flex;
+      flex-direction: column;
+      h1 {
+        font-size: 26px;
+        line-height: 38px;
+        text-align: center;
+        text-transform: uppercase;
+        color: $green;
+        font-weight: 300;
+        span {
+          color: $white;
+        }
+      }
+      p {
+        padding: 16px 0px 33px 0px;
+        font-size: 15px;
+        line-height: 25px;
+        text-align: center;
+        color: $silver;
+        font-weight: 300;
+      }
+      .form {
+        order: 1;
+        .inputGroup {
+          padding: 48px 0px 94px 0px;
+          display: flex;
+          flex-direction: column;
+          align-items: left;
+          input,
+          button {
+            border-radius: 28px;
+            border: none;
+            width: 100%;
+            font-size: 14px;
+            line-height: 28px;
+            font-weight: 700;
+          }
+          input {
+            margin-bottom: 16px;
+            padding: 9px 0px 9px 32px;
+            background-color: $darkGray;
+            color: $fontInput;
+            &:focus {
+              outline-style: none;
+            }
+          }
+          button {
+            color: $black;
+            background-color: $green;
+            padding: 9px 0px;
+          }
+          .error {
+            color: $warning;
+            font-size: 12px;
+            line-height: 14px;
+            padding: 8px 0px 22px 20px;
+          }
+        }
+      }
+      .socialMedia {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        img {
+          height: 17px;
+          color: $lightGray;
+          opacity: 0.5;
+        }
+        .google {
+          height: 11px;
+        }
+        .pocket {
+          height: 15px;
+        }
+      }
+    }
+  }
+  .bgDots {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
Ova losa praksa sa arhitekturom se ponavlja. Media query koji se tice body-ja treba biti nestovan unutar body selektora:
`
body {
    ...
    @media only screen and (min-width: 768px) {...}
    ...
}
`

i ovako za svaku klasu ili selektor, gde god imas responsive kod. Tezimo da izbegnemo arhitekturu fajla gde imamo desktopCss, tabletCss, mobileCss...